### PR TITLE
LPS-160238 Move OAuth 2 tester app

### PR DESCRIPTION
**Pull request by stian-sigvartsen**
[LPS-160238](https://issues.liferay.com/browse/LPS-160238)

### CODE REVIEW

**Comment by stian-sigvartsen** [comment]
@gaborlovasdotcom this is proving very stubborn

**Comment by gaborlovasdotcom** [comment]
ci:stop:security

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by jkappler** [comment]
@stian-sigvartsen, @gaborlovasdotcom, see https://github.com/liferay-appsec/liferay-portal/pull/794

**Comment by stian-sigvartsen** [comment]
@gaborlovasdotcom I have cherry-picked your 454be9d commit from liferay-appsec/liferay-portal/pull/785 and modified it for the new oauth 2 tester app's path. Do we need anything else?

**Comment by gaborlovasdotcom** [comment]
@stian-sigvartsen , That's should be enough, thank you!

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
@stian-sigvartsen , I checked the functional test errors, it's safe to forward as now we could work on the test fixes in a follow-up PR.
[Testray](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1943214795&testrayRunId=1943214830)

**Comment by stian-sigvartsen** [comment]
ci:forward

File: modules/apps/oauth2-provider/oauth2-provider-client/src/main/resources/META-INF/resources/liferay-oauth2-tester/auth.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
+/* eslint-disable */
```
Comment (gaborlovasdotcom) [inline modules/apps/oauth2-provider/oauth2-provider-client/src/main/resources/META-INF/resources/liferay-oauth2-tester/auth.js:1]: We need to remove this parameter and fixup source formatting issues.

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:relevant

**Comment by oliverkecskemety** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:relevant

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2

**Comment by gaborlovasdotcom** [comment]
ci:test:relevant

**Comment by gaborlovasdotcom** [comment]
ci:test:stable

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
ci:test:relevant

**Comment by gaborlovasdotcom** [comment]
Forwarded directly in https://github.com/brianchandotcom/liferay-portal/pull/122024

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
This PR has been forwarded directly to Brian https://github.com/brianchandotcom/liferay-portal/pull/122298

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:outh2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by oliverkecskemety** [comment]
ci:test:oauth2:nocompile

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:stop

**Comment by oliverkecskemety** [comment]
ci:test:oauth2:nocompile

**Comment by oliverkecskemety** [comment]
cI:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:stable

**Comment by gaborlovasdotcom** [comment]
ci:test:relevant

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:stable

**Comment by gaborlovasdotcom** [comment]
ci:test:relevant

**Comment by oliverkecskemety** [comment]
ci:test:relevant

**Comment by oliverkecskemety** [comment]
ci:test:relevant

**Comment by gaborlovasdotcom** [comment]
ci:test:stable

**Comment by gaborlovasdotcom** [comment]
ci:test:relevant

**Comment by oliverkecskemety** [comment]
ci:test:relevant

**Comment by oliverkecskemety** [comment]
Forwarded to https://github.com/brianchandotcom/liferay-portal/pull/123638

**Pull request closed without merging**